tests/test_simultaneous_initiate.c

```diff
@@ -311,6 +322,17 @@ START_TEST(test_lost_simultaneous_initiate)
     result = session_builder_process_pre_key_bundle(bob_session_builder, alice_pre_key_bundle);
```

Comment:
Same thing here. There's another `session_builder_process_pre_key_bundle` function before this one